docs/src/references/tedge-cumulocity-proxy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Thin Edge Cumulocity HTTP Proxy
+tags: [ Reference, HTTP, Cumulocity ]
+sidebar_position: 12
+---
+
+# Thin Edge Cumulocity Proxy
+
+The `tedge-mapper` (when running in `c8y` mode) hosts a proxy server to access the Cumulocity HTTP API as the thin-edge
+device. It automatically handles authenticating with a JWT, avoiding the need for clients to support MQTT to retrieve
```

Comment:
```suggestion
device.
It automatically handles authenticating with a JWT, avoiding the need for clients to support MQTT to retrieve
```
We follow a markdown practice where each sentence starts from a new line and not as a continuation of the previous one. This helps while viewing the diff when changes are made to a few lines in future. Otherwise we'll always end up messing the line wrapping every time we make a change. Long sentences can be broken into multiple lines.